app/templates/java.xml

```diff
@@ -2,12 +2,20 @@
 <job xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="urn:proactive:jobdescriptor:3.12"
 	xsi:schemaLocation="urn:proactive:jobdescriptor:3.12 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.12/schedulerjob.xsd"
 	name="Java_Job">
+	<description>
+		A workflow that executes Java classes.
```

Comment:
Add <![CDATA[ ... ]]> to the description tag. It is missing, right?

Reply:
It is there.

Reply:
I mean CDATA